.github/workflows/testing.yml

```diff
@@ -14,26 +14,31 @@ jobs:
       fail-fast: false
       matrix:
         php-versions: ['7.3', '7.4', '8.0', '8.1']
+        # Keep testing Drupal 9 untill 6 months after it got unsupported, so
+        # untill May 1st 2024.
         drupal-core: ['9.5.x']
         phpstan: ['0']
         include:
           # Extra runs to also test on latest Drupal 10.
           - php-versions: '8.1'
-            drupal-core: '10.1.x'
+            drupal-core: '10.2.x'
             phpstan: '0'
-          # We only need to run PHPStan once on the latest PHP version.
           - php-versions: '8.2'
-            drupal-core: '10.1.x'
+            drupal-core: '10.2.x'
+            phpstan: '0'
+          # We only need to run PHPStan once on the latest PHP version.
```

Comment:
@klausi I usually recommend running PHPStan on the lowest supported PHP version rather than the highest since it prevents people from accidentally using syntax of newer PHP versions that would break PHP compatibility.

I see that's already different from what we did, but we might want to consider changing it.

Reply:
hm, but does phpstan check for PHP version syntax? I usually like to run it on the newest version so that we make our code as forward compatible as possible. PHP syntax errors are then hopefully caught by running the tests on the older PHP versions.